src/test/regress/multi_schedule

```diff
@@ -14,6 +14,7 @@ test: multi_shard_update_delete recursive_dml_with_different_planners_executors
 test: insert_select_repartition window_functions dml_recursive multi_insert_select_window
 test: multi_insert_select_conflict citus_table_triggers
 test: multi_row_insert insert_select_into_local_table multi_create_table_new_features
+test: multi_agg_approximate_distinct
```

Comment:
The reason for this change is because multi_agg_approximate_distinct doesn't have its own schema(so it is not isolated) and this is the fastest/easiest way to accomplish having the necessary tables.